satellite/payments/stripecoinpayments/database.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package stripecoinpayments
+
+import (
+	"context"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+)
+
+// StripeCustomers is interface for working with stripe customers table
+type StripeCustomers interface {
```

Comment:
You can name this as `Customers` otherwise the full name is `stripecoinpayments.StripeCustomers` notice the duplicated `Stripe`